x-pack/test/apm_api_integration/basic/tests/settings/agent_configuration.ts

```diff
@@ -185,7 +185,7 @@ export default function agentConfigurationTests({ getService }: FtrProviderConte
           // ensure that `createConfiguration` throws
           expect(true).to.be(false);
         } catch (e) {
-          expect(e.res.statusCode).to.be(404);
+          expect(e.res.statusCode).to.be(403);
```

Comment:
Yaaay! Returning 403 when upon access denied instead of 404 is going to make debugging much easier!